### DISCUSSION TITLE
How can i use it in django projects and use it with django's jwt

### DISCUSSION THREAD

**Discussion by hzjux001** [PostgREST/postgrest#1777] [Q&A] [answered]
How can i use it in django projects and use it with django's jwt

**Comment by steve-chavez** [comment, marked as answer]
Hey @hzjux001,

PostgREST requirements for the JWT are:

- It must contain a `role` claim that maps to a PostgreSQL role([ref](https://postgrest.org/en/v7.0.0/auth.html#authentication-sequence))
- It should be able to decode it with the [jwt-secret](https://postgrest.org/en/v7.0.0/configuration.html#jwt-secret)

So if django(or any other framework) produces the JWT, then it should share its secret with PostgREST plus construct the JWT with a `role` claim.
